src/DynamoCore/Models/RecordableCommands.cs

```diff
@@ -1315,7 +1315,6 @@ public override string ToString()
         }
 
         [DataContract]
-        [Obsolete("Node to Code not enabled, API subject to change.")]
         public class ConvertNodesToCodeCommand : NodeSpecificRecordableCommand
         {
             #region Public Class Methods
```

Comment:
Since you're relying on `DynamoSelection` for the selection set, the argument `nodeId` is no longer needed.